producer.go

```diff
@@ -98,8 +97,8 @@ func (p *Producer) consumeDeliveries() {
 				atomic.AddUint64(&stats.producerErr, 1)
 			}
 		} else {
-			p.log.Printf("Error consuming delivery event: Unknown event type (%s)", ev)
-			atomic.AddUint64(&stats.producerErr, 1)
+			p.log.Printf("Connection error (%s)", ev)
+			atomic.AddUint64(&stats.connectionErr, 1)
```

Comment:
We want to ignore "Connection reset by peer" events (we don't want those incrementing the counter).